README.md

```diff
@@ -47,7 +47,11 @@ Options:
 
   --sample-rate INTEGER  Perform resampling only to this sample rate. By
                          default we resample to 16000, 22050, 44100, 48000.
+  
+  --small       BOOL     If True, the task will run on a small-version of the 
+                         data.
```

Comment:
False is the default if this is unspecified right?

Can you just make this argument a flag, either it exists or it doesn't, and no bool value is needed?